packages/ember-glimmer/lib/helpers/component.js

```diff
@@ -66,20 +66,36 @@ import { assert } from 'ember-metal';
 
   ```handlebars
   {{yield (hash
-      nameInput=(component "my-input-component" value=model.name placeholder="First Name"))}}
+    nameInput=(component "my-input-component" value=model.name placeholder="First Name")
+  )}}
   ```
 
-  The following snippet:
+  When yielding the component via the `hash` helper, the component is invocked directly.
+  See the following snippet:
 
   ```
   {{#person-form as |form|}}
-    {{component form.nameInput placeholder="Username"}}
+    {{form.nameInput placeholder="Username"}}
   {{/person-form}}
   ```
 
-  would output an input whose value is already bound to `model.name` and `placeholder`
+  Which outputs an input whose value is already bound to `model.name` and `placeholder`
   is "Username".
 
+  When yielding the component without the hash helper use the `component` helper:
+
+  ```handlebars
+  {{yield
+    nameInput=(component "my-input-component" value=model.name placeholder="First Name")
```

Comment:
I think this is invalid syntax? Should just be {{yield (component ....)}} I think